internal/controllers/reconciliation/overrides_test.go

```diff
@@ -539,3 +539,97 @@ func TestOverrideContainerResources(t *testing.T) {
 	assert.True(t, deploy.Spec.Template.Spec.Containers[0].Resources.Limits["cpu"].Equal(resource.MustParse("20m")))
 	assert.True(t, deploy.Spec.Template.Spec.Containers[0].Resources.Requests["cpu"].Equal(resource.MustParse("5m")))
 }
+
+func TestOverrideReplaceAnnotation(t *testing.T) {
```

Comment:
Cool, so this essentially creates a resource through reconcile, make an update from an external client that causes the resource to be replaced, Eno replaces the resource thus removing the external client changes (including the one that caused the replace in the first place), make an external update again that is now not replaced?

Reply:
Yep! 💯